recipes/betse/meta.yaml

```diff
@@ -0,0 +1,86 @@
+{% set name = "betse" %}
+{% set version = "0.7.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "845896f4c30b3970d5bd9501e0ca754eef5af0d742de76103f66c8c8c58e4eca" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
```

Comment:
remove single quotes

Reply:
Ah, more `conda skeleton pip` inanity. I'll get this cleaned up post-haste. Thanks, @carlodri! You two are *really* on the ball here. :+1: